packages/frontend/src/components/Target.vue

```diff
@@ -14,16 +14,36 @@
     <br /><br />
     <span class="subtitle-color-field">Select Cloud Foundry Organization </span><span class="text-danger">*</span><br />
     <vscode-dropdown class="mt-8 dropdown" position="below" @change="changeOrg">
-      <vscode-option v-for="o in orgs" :key="o.guid" :value="o.guid" :selected="o.selected">{{
-        o.label
-      }}</vscode-option>
+      <vscode-option
+        v-for="o in orgs"
+        :key="o.guid"
+        :value="o.guid"
+        :selected="o.selected"
+        v-tooltip="{
+          content: o.label,
+          placement: 'right',
+          class: 'tooltip-custom',
+          size: '10%',
```

Comment:
consider to move [size && placement] into class 'tooltip-custom' css definition - it makes code clear

Reply:
These properties should be in the declaration and not in the CSS file,
If you try to put it in the style section you will see the error: "Unknown property: 'placement'css(unknownProperties)"